src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp

```diff
@@ -335,6 +335,45 @@ class jit_select_emitter : public jit_emitter {
     void emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const;
 };
 
+class jit_soft_sign_emitter : public jit_emitter {
```

Comment:
I expect `SoftPlus` declaration here. `SoftSign` should be here: https://github.com/openvinotoolkit/openvino/issues/24114.